spec/uploadcare/client/addons_client_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module Uploadcare
+  module Client
+    RSpec.describe AddonsClient do
+      subject { AddonsClient.new }
+
+      describe 'uc_clamav_virus_scan' do
+        it 'scan the file for viruses' do
+          VCR.use_cassette('uc_clamav_virus_scan') do
+            uuid = 'ff4d3d37-4de0-4f6d-a7db-8cdabe7fc768'
+            params = { purge_infected: true }
+            response = subject.uc_clamav_virus_scan(uuid, params)
+            expect(response.success).to eq({ request_id: '34abf037-5384-4e38-bad4-97dd48e79acd' })
+          end
+        end
+      end
+
+      describe 'uc_clamav_virus_scan_status' do
+        it 'checking the status of a virus scanned file' do
+          VCR.use_cassette('uc_clamav_virus_scan_status') do
+            uuid = '34abf037-5384-4e38-bad4-97dd48e79acd'
+            response = subject.uc_clamav_virus_scan_status(uuid)
+            expect(response.success).to eq({ status: 'done' })
+          end
+        end
+      end
+
+      describe 'ws_rekognition_detect_labels' do
+        it 'execute aws rekognition' do
```

Comment:
здесь и ниже есть такое же

```suggestion
        it 'executes aws rekognition' do
```